pkg/client/util/util.go

```diff
@@ -71,3 +72,22 @@ func OSArchFromTag(tag string) (api.OS, api.Architecture) {
 
 	return os, arch
 }
+
+func TagMaptoList(tags map[string]api.ImageTag) []api.ImageTag {
+	taglist := make([]api.ImageTag, 0, len(tags))
+	for _, tag := range tags {
+		taglist = append(taglist, tag)
+	}
+	return taglist
+}
+
+func FindParentTags(tags map[string]api.ImageTag, tag string, current *api.ImageTag) {
+	// Already exists — add as child
+	if parent, exists := tags[tag]; exists {
+		parent.Children = append(parent.Children, current)
+		tags[tag] = parent
```

Comment:
could you do `tags[tag].Children = append(tags[tag].Children, current)`?

Reply:
```./util.go:87:3: cannot assign to struct field tags[tag].Children in map```